src/index.js

```diff
@@ -437,8 +440,8 @@ Dropzone.propTypes = {
   disableClick: PropTypes.bool,
 
   /**
- * Enable/disable the dropzone entirely
- */
+   * Enable/disable the dropzone entirely
+   */
```

Comment:
Can you format these back? seems a bit quirky out of line w/ the prop. Sorry just a nitpick

Reply:
sure 😅

Reply:
We don't need to format. prettier should take care of this. Did lint-staged run?

Reply:
Also, I agree this looks better now so let's keep it, please.